IS7024/Pages/LudlowGarage.cshtml

```diff
@@ -6,72 +6,69 @@
-            string[] tars2 = tars[1].Split(':');
-            string mhour = tars2[0];
-            string mmin = tars2[1];
-
-            var newhour = Convert.ToInt32(mhour);
-            int realhour = newhour - 12;
-            string starttime = realhour + ":" + mmin + " PM";
-
-
-                     
+<div class="text-center">
+    <h1 class="display-4">Upcoming Concerts at the Ludlow Garage</h1>
+    <p>Cincinnati, Ohio</p>
+    <hr />
+    <table border="1" width="90%">
+        <tr>
+            <th>Date</th>
+            <th>Artist</th>
+            <th>Status</th>
+            <th>Start Time</th>
+            <th>Sunset Time</th>
+            <th>Buy tickets</th>
+        </tr>
+        @{
+
+            foreach (QuickType.Event eventss in events)
             {
-               
-
-                    <tr>
-                        <td>@day/@month/@yearclean</td>
-                        <td>@eventss.Performance[0].Artist.DisplayName</td>
-                        <td>@eventss.Status</td>
-                        <td>@starttime</td>
-                        <td>@sun</td>
-                        <td><a href="@eventss.Uri" target="_blank">Get Tickets</a></td>
-
-                       <td></td>
-                    </tr>
-            }
-        }
 
+                string date = eventss.Start.Date.ToString();
+                string[] dars = date.Split('/');
+                string day = dars[0];
+                string month = dars[1];
+                string year = dars[2];
+                string[] year1 = year.Split(' ');
+                string yearclean = year1[0];
+
+
+                if (eventss.Start.Datetime != null)
+
+                {
+                    string[] tars = eventss.Start.Datetime.Split('T');
+                    string[] tars2 = tars[1].Split(':');
+                    string mhour = tars2[0];
+                    string mmin = tars2[1];
+
+                    var newhour = Convert.ToInt32(mhour);
+                    int realhour = newhour - 12;
+                    string starttime = realhour + ":" + mmin + " PM";
 
-    }
 
 
+                    {
 
 
+                        <tr>
+                            <td>@day/@month/@yearclean</td>
+                            <td>@eventss.Performance[0].Artist.DisplayName</td>
+                            <td>@eventss.Status</td>
+                            <td>@starttime</td>
+                            <td>@sun</td>
+                            <td><a href="@eventss.Uri" target="_blank">Get Tickets</a></td>
 
-    
-        <img src="https://raw.githubusercontent.com/huberr3/IS7024/master/powered-by-songkick-pink.png" width="171" height="60">
-    
-    }
+                            <td></td>
+                        </tr>
+                    }
+                }
+
+
+            }
+
+
+        }
+    </table>
+    <img src="https://raw.githubusercontent.com/huberr3/IS7024/master/powered-by-songkick-pink.png" width="171" height="60">
 
+</div>
```

Comment:
Added end tag for element 'div" to make code more readable and also to ensure it is future development proof